src/elisp/treemacs-workspaces.el

```diff
@@ -138,18 +138,26 @@ Will return `point-min' if there is no next project."
   (inline-quote
    (setq treemacs--project-positions (make-hash-table :test #'equal :size 20))))
 
+(define-inline treemacs-project->key (project)
```

Comment:
This happens with root-level extensions [here](https://github.com/Alexander-Miller/treemacs/blob/master/src/elisp/treemacs-extensions.el#L443) and [here](https://github.com/Alexander-Miller/treemacs/blob/master/src/elisp/treemacs-extensions.el#L465) - it might be considered a bug though...